peakina/readers/csv.py

```diff
@@ -0,0 +1,79 @@
+"""
+Module to add csv support
+"""
+from typing import Any, Dict, List, Optional
+
+import pandas as pd
+
+# The chunksize value for previews
+PREVIEW_CHUNK_SIZE = 1024
+
+
+def _extract_columns(filepath: str, encoding: str, sep: str) -> List[str]:
+    with open(filepath, buffering=10000, encoding=encoding) as ff:
+        return ff.readline().replace("\n", "").split(sep)
```

Comment:
```suggestion
        return ff.readline().rstrip("\n").split(sep)
```
Probably safer and more explicit

Reply:
> Why did you modify the http fetcher? How is it related to csv and excel? thinking

because openpyxl and xlrd are using the file extension to know to know if this is an xls or an xlsx file. It is not required by this PR thouhgh, so it can be removed for the time being.